src/test/resources/com/tresys/magvar/magvar.tdml

```diff
@@ -0,0 +1,742 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<testSuite suiteName="Namespaces"
+  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/" 
+  xmlns:tdml="http://www.ibm.com/xmlns/dfdl/testData"
+  defaultRoundTrip="onePass">
+
+	<tdml:parserTestCase name="magvar_test"
+		model="com/tresys/magvar/xsd/magvar.dfdl.xsd">
+		<tdml:document>
+			<tdml:documentPart type="file">WMM.COF</tdml:documentPart>
+		</tdml:document>
+		<tdml:infoset>
+			<tdml:dfdlInfoset>
+				<magvar>

```

Comment:
Too big infoset for a inline test in a TDML file. Put infoset into separate file please.